### PR TITLE
[stable10] Run the license script

### PR DESCRIPTION
**Pull request by nickvergessen**
Similar to #507

### CODE REVIEW

**Comment by MorrisJobke** [comment]
👍

File: AUTHORS [hunk 1]
```diff
@@ -145,6 +150,8 @@ ownCloud is written by:
  - Sebastian Döll <sebastian.doell@libasys.de>
  - Senorsen <senorsen.zhang@gmail.com>
  - Serge Martin <edb@sigluy.net>
+ - Sergio Bertolin <sbertolin@solidgear.es>
+ - Sergio Bertolín <sbertolin@solidgear.es>
```
Comment (rullzer) [inline AUTHORS]: double?

File: apps/admin_audit/appinfo/app.php [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <?php
 /**
+
```
Comment (rullzer) [inline apps/admin_audit/appinfo/app.php]: newline?

File: AUTHORS (hunk 1 above)
Comment (nickvergessen) [inline AUTHORS]: `i` vs. `´i`?

Should add a map entry later, nbut nothing to fix here.

Comment (rullzer) [inline AUTHORS]: ah right.

**Comment by rullzer** [comment]
:+1:

**Comment by rullzer** [comment]
Needs rebase

**Comment by schiessle** [comment]
:+1:

**Pull request merged**